### ISSUE TITLE
crashing when baritone renders something

### ISSUE THREAD

**Issue by UltraBlackLinux** [xCollateral/VulkanMod#157]
Hey there,
I just noticed that minecraft is crashing whenever baritone tries to render anything:
```java
[09:30:30] [Render thread/INFO]: [CHAT] [Baritone] Going to: GoalXZ{x=-28,z=-33}
[09:30:30] [Render thread/ERROR]: Unreported exception thrown!
java.lang.RuntimeException: unknown drawType
	at net.vulkanmod.vulkan.Drawer.draw(Drawer.java:105) ~[VulkanMod_1.19.2-0.2.0+1.jar:?]
	at net.minecraft.class_286.method_43433(class_286.java:550) ~[client-intermediary.jar:?]
	at net.minecraft.class_289.method_1350(class_289.java:27) ~[client-intermediary.jar:?]
	at baritone.utils.PathRenderer.drawDankLitGoalBox(PathRenderer.java:343) ~[baritone-fabric-unoptimized-1.19.2-SNAPSHOT.jar:?]
	at baritone.utils.PathRenderer.render(PathRenderer.java:101) ~[baritone-fabric-unoptimized-1.19.2-SNAPSHOT.jar:?]
	at baritone.behavior.PathingBehavior.onRenderPass(PathingBehavior.java:558) ~[baritone-fabric-unoptimized-1.19.2-SNAPSHOT.jar:?]
	at baritone.event.GameEventHandler.lambda$onRenderPass$6(GameEventHandler.java:108) ~[baritone-fabric-unoptimized-1.19.2-SNAPSHOT.jar:?]
	at java.util.concurrent.CopyOnWriteArrayList.forEach(CopyOnWriteArrayList.java:807) ~[?:?]
	at baritone.event.GameEventHandler.onRenderPass(GameEventHandler.java:108) ~[baritone-fabric-unoptimized-1.19.2-SNAPSHOT.jar:?]
	at net.minecraft.class_761.handler$bad000$onStartHand(class_761.java:4984) ~[client-intermediary.jar:?]
	at net.minecraft.class_761.method_22710(class_761.java:1554) ~[client-intermediary.jar:?]
	at net.minecraft.class_757.method_3188(class_757.java:1039) ~[client-intermediary.jar:?]
	at net.minecraft.class_757.method_3192(class_757.java:816) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1523(class_310.java:1177) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1514(class_310.java:768) [client-intermediary.jar:?]
	at net.minecraft.client.main.Main.method_44604(Main.java:244) [client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:51) [client-intermediary.jar:?]
	at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:578) ~[?:?]
	at org.quiltmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:423) [quilt-loader-0.17.7.jar:?]
	at org.quiltmc.loader.impl.launch.knot.Knot.launch(Knot.java:76) [quilt-loader-0.17.7.jar:?]
	at org.quiltmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:24) [quilt-loader-0.17.7.jar:?]
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:88) [NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:126) [NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:71) [NewLaunch.jar:?]](https://github.com/xCollateral/VulkanMod)
```
Can you fix this?
Thanks!

Edit: Edited error message to account for 0.2.1 changes

**Comment by UltraBlackLinux** [comment]
I can still reproduce this with the new 0.2.1 version

**Comment by nm17** [comment]
Also can reproduce on 0.2.5 on 1.19.4.

Full log: https://mclo.gs/wD8YPEX

**Comment by ZeroMemes** [comment]
Will be fixed by next major release of Baritone for 1.18.2 and 1.19.4 (https://github.com/cabaletta/baritone/pull/4059)

**Comment by UltraBlackLinux** [comment]
oh awesome, thank you!
